pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -89,3 +89,20 @@ def arange(start, stop, step):
         return torch.arange(start, stop, step, dtype=dtype)
 
     return arange
+
+
+@pytorch_funcify.register(Join)
+def jax_funcify_Join(op, **kwargs):
+    def join(axis, *tensors):
+        # tensors could also be tuples, and in this case they don't have a ndim
+        tensors = [torch.tensor(tensor) for tensor in tensors]
+        view = op.view
```

Comment:
We wast to remove this kwarg, so you can just raise NotImplementedErrror if the Op has it set in the outer dispatch function.